### ISSUE TITLE
Configuring a changelog class instead of a package does not result in executed changelog

### ISSUE THREAD

**Issue by bvklingeren** [cloudyrock/mongock#233]
I'm using mongock-spring-v5 v4.1.12 with the mongodb-springdata-v3-driver v4.1.12.

I have a ChangeLog class with correct annotations:

```
package github.changelogs;

@ChangeLog
public class DbChangeLog {

    @ChangeSet(id = "changeset1", order = "001", author = "me")
    public void changeSet1(MongockTemplate mongoTemplate) {
    }
}
```

When I run de changelog (in a test) like so:

```
        MongockSpring5.builder()
                .setDriver(SpringDataMongo3Driver.withDefaultLock(mongoTemplate))
                .addChangeLogClass(DbChangeLog.class)
                .setSpringContext(applicationContext)
                .buildApplicationRunner()
                .execute();
```

The changelog is not executed.

When I run the changelog with a package name it does run:

```
        MongockSpring5.builder()
                .setDriver(SpringDataMongo3Driver.withDefaultLock(mongoTemplate))
                .addChangeLogsScanPackage(DbChangeLog.class.getPackage().getName())
                .setSpringContext(applicationContext)
                .buildApplicationRunner()
                .execute();
```

I expected that running Mongock with the classname of the changelog class (the first case) would also work.

**Comment by dieppa** [comment]
Hello @bvklingeren,

thanks. That's a bug we found a couple of days ago, but we've had some other priorities. But no worries, we'll have it fixed by today.

Cheers

**Comment by dieppa** [comment]
Fixed in version 4.1.13.
 @bvklingeren , can you give a try, please?
